Piwik.Tracker/PiwikTracker.cs

```diff
@@ -721,11 +742,11 @@ public TrackingResponse DoTrackEcommerceCartUpdate(double grandTotal)
         /// Response
         /// </returns>
         /// <exception cref="System.InvalidOperationException">Error: you must call the function DoTrackPageView or DoTrackGoal from this class, before calling this method DoBulkTrack()</exception>
-        public TrackingResponse DoBulkTrack()
+        public async Task<TrackingResponse> BulkTrackAsync()

```

Comment:
While [this explanation](https://ponyfoo.com/articles/understanding-javascript-async-await) is dealing with JavaScript, the underlying way async await works is the same. (C# doesn't yet provide generator, so it's not so good to explain in C#)